### DISCUSSION TITLE
re-execute all failed runs?

### DISCUSSION THREAD

**Discussion by ycwang** [dagster-io/dagster#5797] [Q&A] [answered]
Hi Dagster team,
Thanks a lot for developing dagster, it feels so natural to use.
I have done a backfill of around 10 years and got around 600 failed runs, it is so tedious to click into every run to re-execute the run from failure....
Is it possible to re-execute all failed runs? either in dagit or in python?

Thanks again,
Tim

**Comment by ycwang** [comment, marked as answer]
Found it! for those who face the same problem, you can lanch the backfill on the "partition" page, and then click "re-execute from failures",  the success runs will be automatically skipped.
thanks team, best and happy holidays!

-Tim

**Reply by sryza** [reply]
That's right. Glad you figured it out @ycwang.
